apis/r/NAMESPACE

```diff
@@ -5,7 +5,9 @@ S3method("[[<-",MappingBase)
 S3method(as.list,MappingBase)
 S3method(length,MappingBase)
 S3method(names,MappingBase)
+export(Collection)
```

Comment:
I have been pondering this yesterday and still think 'Collection' may be a little too generic and hence _possibly_ at the risk of a NAMESPACE clash.  That said I don't yet know or think of another package having such an identifier, and maybe being uppercase will save us.